build.gradle

```diff
@@ -35,6 +35,10 @@ buildScan {
     licenseAgree = 'yes'
 }
 
+String hailVersion = "0.1"
+String[] supportedSparkVersions = ["2.0.2", "2.1.0"]
+String gitHash = exec {commandLine 'git', 'rev-parse', '--short', 'HEAD'}
+
 String sparkVersion = System.getProperty("spark.version","2.0.2")
 
 if (!(sparkVersion ==~ /^2\..*/))
```

Comment:
no, because the cdh deployments have weird names and we don't want to have to keep track of those explicitly.